geopandas/base.py

```diff
@@ -432,6 +432,24 @@ def within(self, other):
         """
         return _binary_op("within", self, other)
 
+    def covers(self, other):
+        """
+        Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
+        each geometry that is entirely covering `other`.
+
+        An object A is said to cover another object B if no points of B lie
+        in the exterior of A.
+
+        See https://lin-ear-th-inking.blogspot.com/2007/06/\
+        subtleties-of-ogc-covers-spatial.html for reference.
```

Comment:
So we can do either

```
        See
        https://lin-ear-th-inking.blogspot.com/2007/06/subtleties-of-ogc-covers-spatial.html
        for reference.
```

(if it's a single "word" without space, the linter won't complain that it is too long, I think)

or either

```
        See https://lin-ear-th-inking.blogspot.com/2007/06/\
subtleties-of-ogc-covers-spatial.html for reference.
```

(the continued line should not be intended in this case, otherwise it would add those spaces in the middle of the url in the actual docstring users see)

Both are a bit ugly, but maybe the first is best (that makes it the easiest to copy the url also when looking at it in the source code)

Reply:
Ah! Sorry about that, I didn't realize the continued line after `\` was indented. I'm not able to build the docs for some reason else it'd be easier to check. Trying the first option.